test/tests/outputs/oversample/oversample_file.i

```diff
@@ -42,7 +42,7 @@
 [Outputs]
   [./exodus]
     type = Exodus
-    refinements = 2
+    refinements = 1
```

Comment:
I understand this better now... And this way actually makes more sense I think.  Closer to what a new user would expect.